.github/workflows/validation-linux.yml

```diff
@@ -2,9 +2,10 @@ name: validation-linux
 on:
   pull_request:
     paths:
-    - 'tests/out/*.spvasm.snap'
-    - 'tests/out/*.glsl.snap'
-    - 'tests/out/*.dot.snap'
+    - 'tests/out/*.spvasm'
```

Comment:
Looks like test infrastructure been broken for a while after https://github.com/gfx-rs/naga/pull/720

Reply:
ahh, thank you for spotting this!